### PR TITLE
core: frontend: do not return 80 if port is unknown

### PR DESCRIPTION
**Pull request by Williangalvani**
should fix the blueos-in-blueos issue

williangalvani/blueos-core:no-port-80

### CODE REVIEW

File: core/frontend/src/views/ExtensionView.vue [hunk 1]
```diff
@@ -27,10 +34,13 @@ export default Vue.extend({
     fullpage(): boolean {
       return this.$route.query.full_page === 'true'
     },
-    port(): number {
+    port(): number | undefined | null {
```
Comment (Williangalvani) [inline core/frontend/src/views/ExtensionView.vue:37]: is it obvious what it means? or should I add a comment?
number = port
undefined = we dont know (still loading)
null = nope

File: core/frontend/src/views/ExtensionView.vue [hunk 2]
```diff
@@ -4,18 +4,25 @@
     :source="service_path"
     :class="fullpage ? 'fullpage' : ''"
   />
+  <SpinningLogo v-else-if="detected_port === undefined" />
```
Comment (patrickelectric) [inline core/frontend/src/views/ExtensionView.vue:7]: can we put v-else-if and v-else in the beggning of the component for better readability ?

Like:
`<v-else page-not-found/>`

**Review by patrickelectric — COMMENTED** [review]
Check comment

File: core/frontend/src/views/ExtensionView.vue (hunk 2 above)
Comment (Williangalvani) [reply core/frontend/src/views/ExtensionView.vue:7]: that doesn't seem to be valid syntax

**Review by patrickelectric — APPROVED** [review]

**Pull request merged**